src/FromFile.jl

```diff
@@ -21,25 +19,27 @@ function from_m(m::Module, path::String, ex::Expr)
 
     if root === Main
         toplevel = __toplevel__
-    elseif isdefined(root, Symbol("#__toplevel__#")) # package
-        toplevel = getfield(root, Symbol("#__toplevel__#"))
     else
-        toplevel = Base.eval(root, :(baremodule $(Symbol("#__toplevel__#"));using Base;end))
+        toplevel_symbol = :__toplevel__
```

Comment:
I think just in case, let's mangle the name with `Symbol("#__toplevel__#")`

Reply:
Done.